File: src/main/org/nlogo/prim/etc/_resetperspective.java
package org.nlogo.prim.etc;

import org.nlogo.nvm.Command;
import org.nlogo.nvm.Context;
import org.nlogo.nvm.Syntax;

public final strictfp class _resetperspective
    extends Command {
  @Override
  public Syntax syntax() {
    return Syntax.commandSyntax("OTPL", true);
  }

  @Override
  public void perform(final Context context) {
    world.observer().resetPerspective();
    context.ip = next;
  }
}

File: src/main/org/nlogo/prim/etc/_resetperspective.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.api.Syntax
import org.nlogo.nvm.{ Command, Context }

class _resetperspective extends Command {
  override def syntax =
    Syntax.commandSyntax("OTPL", true)
  override def perform(context: Context) {
    world.observer.resetPerspective()
    context.ip = next
  }
}
